vidify/gui/components.py

```diff
@@ -178,15 +180,27 @@ def load_players(self, saved_player: Optional[str]) -> None:
 
         # The cards are inside a group so that their selection is exclusive.
         self.player_group = QButtonGroup()
+        # The disabled players will always be at the end of the layout, so
+        # they're saved in a list to add them later.
+        disabled = []
         for player in PlayerData:
+            available = CURRENT_PLATFORM in player.platforms
+            enabled = player.installed
             selected = False if saved_player is None \
                 else player.name == saved_player.upper()
             card = Card(player.name, player.short_name, player.description,
-                        player.icon, selected=selected)
-            self.player_layout.addWidget(card)
+                        player.icon, enabled, selected)
+            if available:
+                if enabled:
+                    self.player_layout.addWidget(card)
+                else:
+                    disabled.append(card)
             self.player_group.addButton(card.button)
-            logging.info("Added Player card: %s (selected=%s)", player.name,
-                         selected)
+            logging.info("Added Player card: %s (enabled=%s, selected=%s)",
+                         player.name, enabled, selected)
+
+            for card in disabled:
```

Comment:
Shouldn't this be outside the `for player in PlayerData` loop?

Reply:
Oopsie, I think I messed up the indentation when copying that from the API section, sorry about that, should be fixed now.